packages/create-sitecore-jss/src/templates/angular/package.json

```diff
@@ -110,6 +110,6 @@
     "npm-run-all": "~4.1.5",
     "protractor": "^7.0.0",
     "ts-node": "~10.9.2",
-    "typescript": "~5.2.2"
+    "typescript": "~5.4.0"
```

Comment:
angular sample will continue using typescript 5.4.0 for compatibility with current Angular version